cli/kata-check.go

```diff
@@ -76,14 +76,11 @@ func getCPUInfo(cpuInfoFile string) (string, error) {
 		return "", err
 	}
 
-	cpus := strings.SplitAfter(text, "\n\n")
-
-	trimmed := strings.TrimSpace(cpus[0])
-	if trimmed == "" {
+	if len(strings.TrimSpace(text)) == 0 {
```

Comment:
This function is now returning *all* details for *all* cpus fwics. This is to accommodate PPC64le which has an unusual `/proc/cpuinfo` - it contains an extra paragraph at the end which includes the `model` details:

- https://github.com/kata-containers/runtime/blob/master/cli/kata-check_data_ppc64le_test.go#L111

I guess there is no harm *per se* in returning all the duplicated information but it still feels a bit odd to me. Assuming this works as expected on other platforms, please can you update the comment for the function as it is now incorrect as it states:

> // getCPUInfo returns details of **the first CPU** read from the specified cpuinfo file

Hi @Pennyzct and @alicefr - Happy 2019! :smile: Please could you take a look from the ARM64 and os390x perspectives please?

Reply:
Hi~ @jodh-intel, Happy 2019! :) ARM64 share the same structure as amd64, all duplicated info. Do we need to make this func arch-dependent??

Reply:
Yep - I agree with both you and @grahamwhaley @Pennyzct - it would be much clearer to have arch-specific functions for this.